.github/workflows/ci.yaml

```diff
@@ -47,7 +47,7 @@ jobs:
     timeout-minutes: 10
     strategy:
       matrix:
-        go-version: [1.14.x, 1.15.x, 1.16.x]
+        go-version: [1.14.x, 1.15.x, 1.16.x, 1.17.x]
```

Comment:
We can remove all former version and only keep 1.17.x in order to speed up CI runs.

Reply:
Done.